client/app.jsx

```diff
@@ -4,6 +4,13 @@ import Detail from './pages/detail';
 import Cart from './components/cart';
 import Navbar from './components/navbar';
 import Checkout from './pages/checkout';
+import CheckingOut from './pages/checkingout';
+import { loadStripe } from '@stripe/stripe-js';
+import { Elements } from '@stripe/react-stripe-js';
+import Stripe from './components/stripe';
+import Confirmation from './pages/confirmation';
+
+const stripePromise = loadStripe('pk_test_51M4tgyL4bDK4Pth0p1jmAqW0MmEtf8VYIWIUKcgV0XhBQ967SOjcRfFvuKCZ1C7enyhP5D1cmqctxHGjYkjTlO2700USk9mYdw');
```

Comment:
We should use `process.env.STRIPE_API_KEY` here so that any other dev can use their own key. you'll need to update your webpack config to include `'STRIPE_API_KEY'`

https://github.com/Learning-Fuze/final-project-instructions/blob/main/code-base.md#webpackconfigjs